### PR TITLE
lumina: update to 1.6.1.

### PR DESCRIPTION
**Pull request by karlgrose**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)

### CODE REVIEW

File: srcpkgs/lumina/patches/checkpass.patch [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
++target.path = $$DESTDIR$${PREFIX}/bin
+ 
+ LIBS     += -lpam
+ 
+ SOURCES += main.c
+ 
+-perms.path = $$DESTDIR$${PREFIX}/sbin
+-perms.extra = "chmod 4555 $$DESTDIR$${PREFIX}/sbin/lumina-checkpass"
++perms.path = $$DESTDIR$${PREFIX}/bin
++perms.extra = "chmod 4555 $(INSTALL_ROOT)$$DESTDIR$${PREFIX}/bin/lumina-checkpass"
```
Comment (ericonr) [inline srcpkgs/lumina/patches/checkpass.patch:17]: Please add an explanation for what the patch does on top of the file

Comment (karlgrose) [reply srcpkgs/lumina/patches/checkpass.patch:17]: The lumina-checkpass binary, which I understand is to be disabled in release 1.6.2 according to comments on the project site (https://github.com/lumina-desktop/lumina/issues/774), was set to install to a hardwired "sbin" location, but that generates a conflict with the pkg-src sanity/lint tests. This patch just forces the build to install to a "bin" path resulting in the pkg-src checks being passed. The need for this patch should disappear with 1.6.2 if the 774 issue cited is to be believed.

Comment (karlgrose) [reply srcpkgs/lumina/patches/checkpass.patch:18]: patch updated with brief comment

**Comment by ericonr** [comment]
Please remember to squash commits as well.

**Comment by ericonr** [comment]
Hi! Did you mean to close this PR?

**Comment by karlgrose** [comment]
Sorry for the noise; closing was an unintended side effect of my git rebase/cleanup efforts. Then I saw that Lumina release 1.6.2 is imminent, so decided to wait for that and start over, having improved my git/workflow skills in the meantime. Thanks for your patience.

**Comment by ericonr** [comment]
No problem. Looking forward to the upcoming PR.

**Pull request closed without merging**